picoquic/sender.c

```diff
@@ -1385,7 +1385,6 @@ size_t picoquic_prepare_mtu_probe(picoquic_cnx_t* cnx,
     size_t length = header_length;
 
     bytes[length++] = picoquic_frame_type_ping;
-    bytes[length++] = 0;
```

Comment:
Please verify this as it seemed superfluous to do the extra assignment.

The semantic change without the extra assignment should just be the case `probe_length - checksum_length - length = 0`. That should have crashed before, now it doesn't assign any 0. I assume that case never happens as it has been checked by the calling function.

Reply:
Yes. The probe is never smaller than the 1200 bytes min MTU size, and thus the issue never arises.